api/src/puzzle/service/puzzleService.ts

```diff
@@ -0,0 +1,48 @@
+import Puzzle, { IPuzzle } from '../model/puzzle'
+import logger from '../../../utils/logger'
+import { ErrorResponse } from '../../common/errorResponse'
+import ProducerService from '../../producer/service/producerService'
+import { Dimension } from '../../common/dimension'
+
+type PuzzleRequest = {
+    title: string
+    producerId: string
+    numberOfPieces: number
+    dimensions: Dimension
+    modelNumber: string
+    releaseDate: Date
+    recommendedAge: string
+}
+
+export type CreateProducerResponse = { puzzleId: string } | ErrorResponse
```

Comment:
Nie wiem czy serwis to jest odpowiednie miejsce do deklarowania typów związanych z requestami i responsami?